### DISCUSSION TITLE
Quasar Build error "Build succeeded"

### DISCUSSION THREAD

**Discussion by willintonvidal** [quasarframework/quasar#10402] [General - Components / Directives / etc]
Good parent, I have a problem trying to build a simple application even if it is the initial one.

I give the command quasar build and it appears this:

![image](https://user-images.githubusercontent.com/5097870/129603497-f1a738af-86a6-4082-9040-429e3d392a97.png)

![image](https://user-images.githubusercontent.com/5097870/129604029-ab650661-bfa3-48bc-96b3-60a85daf16f2.png)
build the blank index
### 
**If someone can help me, thank you very much, since I was browsing and I did not find anything**

**Comment by pdanpdan** [comment]
Can you please read and make a resume of what you understand from the last lines you posted in the first screenshot? The ones starting with Tip:

**Reply by willintonvidal** [reply]
Thanks!
